template/_dynamic_files.jinja2

```diff
@@ -1083,15 +1083,11 @@ To check if your ontology uses this method, check src/ontology/{{ project.id }}-
 
 If your ontology uses Base Module approach, please use the following steps: 
 
-Start by declaring the term to be imported in Protege:
+To check if your ontology uses this method, check src/ontology/{{ project.id }}-odk.yaml to see if `use_base_merging: TRUE` is declared under `import_group`
 
-1. Open your ontology (edit file) in Protege (5.5+).
-1. Select 'owl:Thing'
-1. Add a new class as usual.
-1. Paste the _full iri_ in the 'Name:' field, for example, http://purl.obolibrary.org/obo/CHEBI_50906.
-1. Click 'OK'
+If your ontology uses Base Module approach, please use the following steps: 
```

Comment:
Lines 1086–1088 are duplicate of lines 1082–1084.

Reply:
whoops, my bad, i accidentally commited on a dirty branch, and was moving things around and missed that. Thanks for spotting :)